docs/release-notes/8.6.asciidoc

```diff
@@ -1,6 +1,16 @@
 [[release-notes-header-8.6.0]]
 == 8.6
 
+[discrete]
+[[release-notes-8.6.1]]
+== 8.6.1
+
+[discrete]
+[[bug-fixes-8.6.1]]
```

Comment:
Can we also add a section that we fixed a bug where 8.6.0 Agents wouldn't correctly send OSQuery to ES?  The 8.6.1 Agent will take care of this.  This is the fix for the known OSQuery issue in 8.6.0

This is the specific ticket: https://github.com/elastic/beats/issues/34250

Reply:
Yep, can do! Did you want to fix worded a particular way in the release notes?

Reply:
Something like the below that is reusing language from the 8.6.0 known issue:

"Fix an issue introduced in 8.6.0 when using the Osquery Manager integration with Elastic Agent, Osquery results aren’t properly written to Elasticsearch and, therefore, cannot be viewed in Kibana.  OSQuery users should upgrade to the 8.6.1 stack and 8.6.1 Agent."

Thoughts?

Reply:
It might be good to simplify parts of the summary for example:

Fixes a bug that prevented Osquery results from being viewed in {kib} when using the Osquery Manager integration with Elastic Agent. Upgrade to {stack} version 8.6.1 and {agent} 8.6.1 to apply this fix to your deployment.

@elastic/security-docs input and suggestions are welcome!

Reply:
> Fixes a bug that prevented Osquery results from being viewed in {kib} when using the Osquery Manager integration with {agent}. Upgrade to {stack} version 8.6.1 and {agent} 8.6.1 to apply this fix to your deployment.

this sounds good to me